packages/terra-image/src/Image.module.scss

```diff
@@ -36,4 +36,32 @@
   .hidden {
     display: none;
   }
+
+  /**
+   * Polyfill 'object-fit-images' for IE requires to use object-fit inside font-family.
+   */
+  .contain {
+    font-family: var(--terra-image-font-family, 'Helvetica Neue', Helvetica, Arial, sans-serif), 'object-fit: contain;';
```

Comment:
Does font family need to be themeable here?

Reply:
No, the font family does not need to be themeable. I have changed it here https://github.com/cerner/terra-core/pull/2494/commits/0ac279fc363761cc6d45b702fc9706bef1be5ac1.